requirements.txt

```diff
@@ -39,7 +39,7 @@ cachetools==5.1.0
     # via
     #   -r requirements.in
     #   notifications-utils
-celery[sqs]==5.2.6
```

Comment:
While we're making the switch, can we also trim out `kombu` and its dependency? https://github.com/celery/celery/blob/master/requirements/extras/sqs.txt

Reply:
It looks like no, since it's also a base requirement: https://github.com/celery/celery/blob/master/requirements/default.txt

Reply:
Ignore that, `kombu` is still a dependency. It's just a slight change in extra extras